CmrUI/Screens/News Feed tab/NewsFeedVC.swift

```diff
@@ -23,75 +17,110 @@ fileprivate struct NewsFeedInsets {
 }
 
 private struct Constants {
+    
+    let kReuseCellID = "FeedNewsCell"
+    let kReuseCellTextID = "FeedNewsTextCell"
+    let kReuseHeader = "FeedNewsSectionHeader"
+    let kImgName = "pict_"
+    let kImgCellHeight: CGFloat = 0.866
+    
     enum SegueID: String {
-        case cardCreationSummary = "DetailedFeedNews"
-        case enterCardAmount = "DescriptionFeedNews"
+        case detailedFeedNews = "DetailedFeedNews"
+        case descriptionFeedNews = "DescriptionFeedNews"
     }
+    
+    let backgroundColor = UIColor(red: 235.0/255.0, green: 235.0/255.0, blue: 235.0/255.0, alpha: 1.0)
 }
 
-fileprivate enum FeedType: Int {
+public enum FeedType: Int {
     case feedImage = 0
     case feedText = 1
     case count = 2
 }
 
 // Will be replaced by mediator for download manager
-fileprivate struct FeedItem {
+public struct FeedItem {
     let image: UIImage?
     let title: String?
     let subtitle: String?
+    let content: String?
     let feedType: FeedType?
+    let time: Date? = nil
 }
 
-class NewsFeedVC: UIViewController, UICollectionViewDelegate, UICollectionViewDataSource {
+class NewsFeedVC: UIViewController, UICollectionViewDelegate, UICollectionViewDataSource, UIViewControllerTransitioningDelegate, UINavigationControllerDelegate {
     
     //Will b moved to the model later, it's a stub for now
     fileprivate var dataSource = [FeedItem]()
     @IBOutlet weak var collectionView: UICollectionView!
+    fileprivate var transitionIndexPath: IndexPath?
+    private var customInteractor : CustomInteractor?
+    private var selectedFrame : CGRect?
+    var zoomTransition: ZoomTransition?
     
     override func viewDidLoad() {
         super.viewDidLoad()
-
         setupAppearance()
         
         dataSource = [
```

Comment:
@vmsavka I think it'd be better to load this from an external class using a protocol (something along the lines of object.dataForItem(item). That class can either use a stub method to provide the data, and when we have a real data source loading (from e.g. URLSession) just redirect the calls.